File: databinding-base/src/main/java/com/hangox/databinding/toast/ToastCake2.java
package com.hangox.databinding.toast;

import android.support.annotation.StringRes;

/**
 * Created With Android Studio
 * User hangox
 * Email liang.hanguang93@gmail.com
 * Date 2017/11/27
 * Time 下午3:59
 */

public interface ToastCake2 {
    void toastLong(String message);

    void toastShort(String message);

    void toastLong(@StringRes int res);

    void toastShort(@StringRes int res);
}

File: databinding-base/src/main/java/com/hangox/starter/toast/ToastCake2.kt
package com.hangox.starter.toast


import androidx.annotation.StringRes

/**
 * Created With Android Studio
 * User hangox
 * Email liang.hanguang93@gmail.com
 * Date 2017/11/27
 * Time 下午3:59
 */

interface ToastCake2 {

    /**
     * 长提示
     */
    fun toastLong(message: String)

    /**
     * 短提示
     */
    fun toastShort(message: String)

    /**
     * 长提示
     */
    fun toastLong(@StringRes res: Int)

    /**
     * 短提示
     */
    fun toastShort(@StringRes res: Int)
}
